actors/miner/tests/terminate_sectors_test.rs

```diff
@@ -79,6 +80,40 @@ fn removes_sector_with_correct_accounting() {
     h.check_state(&rt);
 }
 
+#[test]
+fn removes_sector_with_without_deals() {
```

Comment:
I would add a 3rd sector with only a verified deal to cover that case as well

Reply:
Done.